### DISCUSSION TITLE
Is it possible not to save automatically ckpt file and the end of training, when using LORA, and only generate it when pressing generate button?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [d8ahazard/sd_dreambooth_extension#536] [Q&A] [answered]
Is it possible not to save automatically ckpt file and the end of training, when using LORA, and only generate it when pressing generate button?

**Comment by Thomas-MMJ** [comment, marked as answer]
Yes, there is a patch to disable the automatic checkpoint generation.  Or just comment out the line in the script that does so.

https://github.com/d8ahazard/sd_dreambooth_extension/pull/532
